frontend/src/components/ButtonIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  FunctionComponent,
+  ReactNode,
+  isValidElement,
+  Children,
+  cloneElement,
+} from 'react'
+import cx from 'classnames'
+
+interface ExplicitProps {
+  /**
+   * Specify whether the icon is before or after the text
+   */
+  placement?: 'before' | 'after'
```

Comment:
Question: since `placement` is optional, what is the behavior it isn't present?

Reply:
The icon wouldn't be sized or positioned properly, which isn't good. I made the prop required!